src/content/changes/highlights/2021-12-10.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 2021-12-10T10:00:00
+title: Highlights for the week ending December 10, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+- [flux-app](https://github.com/giantswarm/flux-app) [v0.8.0](https://github.com/giantswarm/flux-app/releases/tag/v0.8.0) includes `helm-controller` fix to reduce memory usage by downgrading `Helm` from `3.7.1` to `3.6.3`.
+- [dex-app](https://github.com/giantswarm/dex-app) [v1.21.0](https://github.com/giantswarm/dex-app/blob/master/CHANGELOG.md#1210---2021-12-09) contains update to the upstream version `v2.30.2`
+
+## User interfaces
+
+- In the Web UI we now display release status in cluster creation form.
```

Comment:
I'm thinking this is not really noteworthy today, as the topic of releases will be big next week.